benchmark/torch/a2c/main.py

```diff
@@ -0,0 +1,49 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
We should keep the same file structure as that in paddle A2C example. Please refer to the [train.py](https://github.com/PaddlePaddle/PARL/blob/60a9d8028013f46aa2804dbef7ec1894ffc47e3e/examples/A2C/train.py) to modify/remove this file.